reference/volumes.html.markerb

```diff
@@ -30,11 +34,15 @@ Explore other options for data storage in [Databases & Storage](/docs/database-s
 
 In a few cases, you can run a single Machine with an attached volume. For example, if you're running an app with a standard SQLite database, or your app is in development and you're not yet worried about downtime.
 
-Many developers use volumes for databases, so if possible, each volume you create for your app is placed in a separate hardware zone by default. Note that having each volume in a separate hardware zone limits the number of volumes your app can have in a region. You can configure this setting with the `--require-unique-zone` option when you run [`fly volumes create`](/docs/flyctl/volumes-create/).
+## Volume placement
+
+You can create volumes at the same time you create Machines, or ahead of time. If you try to create new Machines through `flyctl scale count`, then `flyctl` will first check if you have any unattached volumes in the region you are creating new Machines in. If it finds unattached volumes, it will try to create the new Machines on the same servers. If it does not find any unattached volumes, then it will create new ones together with the Machines. You can also create unattached volumes through [`fly volumes create`](/docs/flyctl/volumes-create/).
```

Comment:
This behaviour is described pretty well here: https://fly.io/docs/apps/scale-count/#scale-an-app-with-volumes.  It might be better to link to it if you think it's belongs in this doc?